bindata/oauth-openshift/deployment.yaml

```diff
@@ -62,6 +62,8 @@ spec:
             - name: https
               containerPort: 6443
               protocol: TCP
+          securityContext:
+            readOnlyRootFilesystem: false # because of the `cp` in args
```

Comment:
@smarterclayton how do you feel about this? Any workaround? Note: we copy the proxy CA to /etc.